.github/workflows/test.yml

```diff
@@ -149,3 +149,22 @@ jobs:
           required-gazebo-distributions: 'harmonic'
       - name: 'Test Gazebo installation'
         run: 'gz sim --versions'
+
+  test_gazebo_install_windows:
+    name: 'Check installation of Gazebo on Windows'
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4.0.2
+        with:
+          node-version: '20.x'
+      - uses: conda-incubator/setup-miniconda@v3
+      - name: 'Check Gazebo installation on Windows runner'
+        uses: ./
+        with:
+          required-gazebo-distributions: 'harmonic'
+      - name: 'Test Gazebo installation'
+        shell: pwsh
+        run: |
+          conda activate
```

Comment:
I was wondering if we can get the `conda activate` automatically called for users in our action so they do not need to run them in the run steps. Maybe adding it to a profile so it is get called automatically on the powershells beind created? https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_profiles?view=powershell-7.4&viewFallbackFrom=powershell-7 .

Reply:
Do you think running a `conda activate` from inside the action will work in this case?